manuscript/book.md

```diff
@@ -82,24 +82,24 @@ const names = ['Bilbo Baggins', 'Gandalf', 'Gollum'];
 const foundName = names.find(name => name.startsWith('B'));
 ```
 
-In both cases I much prefer versions with array methods than with `for` loops. They are shorter and we’re not wasting half the code on iteration mechanics.
+In both cases, I much prefer versions with array methods compared to `for` loops. They are shorter and we’re not bloating the code with iteration mechanics.
 
 ### Implied semantics of array methods
 
 Array methods aren’t just shorter and more readable; each method has its own clear semantic:
 
-- `.map()` says we’re transforming an array into another array with the same number of elements;
-- `.find()` says we’re _finding_ a single element in an array;
-- `.some()` says we’re testing that the condition is true for _some_ array elements;
-- `.every()` says we’re testing that the condition is true for _every_ array element.
+- `.map()` implies we’re transforming an array into another array with the same number of elements;
+- `.find()` implies we’re _finding_ a single element in an array;
+- `.some()` implies we’re testing that the condition is true for _some_ elements of the array;
+- `.every()` implies we’re testing that the condition is true for _every_ element of the array.
 
 Traditional loops don’t help with understanding what the code is doing until you read the whole thing.
 
 We’re separating the “what” (our data) from the “how” (how to loop over it). More than that, with array methods we only need to worry about our data, which we’re passing in as a callback function.
 
-When you use array methods for all simple cases, traditional loops signal to the code reader that something unusual is going on. And that’s good: you can reserve brain resources for better understanding the unusual, more complex cases.
+Using array methods for simple cases and leaving array loops for complicated parts helps the reviewer easily identify the unusual parts of the code. This helps the reviewer concentrate on the complex part of the code.
```

Comment:
⌛️ I'll need to think about this. I don't like the changes but maybe I can improve the original.

Reply:
```suggestion
When all simple cases are covered by array methods, every time you see a traditional loop, you know that something unusual is going on. And that’s good: less chances you’ll miss a bug during code review.


```